lib/auto-languageclient.js

```diff
@@ -270,16 +269,6 @@ export default class AutoLanguageClient {
     });
   }
 
-  handleSpawnFailure(err: any): void {
```

Comment:
I think we still need this to report when starting a language server fails to the user.

Reply:
How does this work though, node processes don't seem to have an `error` event as the previous (unused) code used to call this method.

Reply:
https://nodejs.org/api/child_process.html#child_process_event_error

Reply:
Ah right thanks, I was looking at `process` rather than child-